includes/theme-compat/register-profile.php

```diff
@@ -0,0 +1,104 @@
+                                <div class="form-field-wrapper">
+                                    <input type="password" name="r_password" placeholder="Repeat Password*" required>
+                                </div><?php } ?>
+                            <?php if (!empty($page_name) && $page_name == 'profile') { ?>
+                                <div class="form-field-wrapper">
+                                    <input type="password"
+                                           name="password"
+                                           placeholder="Password">
+                                </div><?php } ?>
+                            <?php if (!empty($page_name) && $page_name == 'profile') { ?>
+                                <div class="form-field-wrapper">
+                                    <input type="password" name="r_password" placeholder="Repeat Password">
+                                </div><?php } ?>
+                            <div class="form-field-wrapper">
+                                <input type="text" name="phone"
+                                       placeholder="Phone Number" <?php if (!empty($_SESSION['input_phone'])) {
+                                    echo 'value="' . $_SESSION['input_phone'] . '" style="color: #3c71fe;" ';
+                                } ?>
+                                    <?php if (!empty($_SESSION['phone'])) {
+                                        echo 'value="' . $_SESSION['phone'] . '" style="color: #3c71fe;" ';
+                                    } ?> >
+                            </div>
+                        </div>
+                        <div class="secondary-container">
+                            <h4 class="form-title">My Company</h4>
+                            <div class="form-field-wrapper">
+                                <input type="text" name="company_name" placeholder="Company Name" <input
+                                        type="text"
+                                        name="phone"
+                                        placeholder="Phone Number" <?php if (!empty($_SESSION['company_name'])) {
+                                    echo 'value="' . $_SESSION['company_name'] . '" style="color: #3c71fe;" ';
+                                } ?>
+                                >
+                            </div>
+                            <div class="form-field-wrapper">
+                                <input type="text" name="company_site"
+                                       placeholder="Company Site" <?php if (!empty($_SESSION['company_site'])) {
+                                    echo 'value="' . $_SESSION['company_site'] . '" style="color: #3c71fe;" ';
+                                } ?>
+                                >
+                            </div>
+                            <div class="form-field-wrapper">
+                                    <textarea name="company_description"
```

Comment:
this is hardly readable, to be honest

Reply:
I made some formatting changes, but I'm not sure it has any effect on better readability.